eclipse-collections-code-generator/src/main/resources/api/set/mutablePrimitiveSet.stg

```diff
@@ -88,6 +88,29 @@ public interface Mutable<name>Set extends Mutable<name>Collection, <name>Set
             return set.toSet().withAll(this);
         }
     }
+
+    /**
+     * Returns the set of all objects that are members of both {@code this} and {@code set}. The intersection of
+     * [1, 2, 3] and [2, 3, 4] is the set [2, 3].
+     *
+     * @since 11.0.
+     */
+    @Override
+    default Mutable<name>Set intersect(<name>Set set)
+    {
+        if (this.size() \< set.size())
+        {
+            Mutable<name>Set mutableSet = this.toSet();
+            mutableSet.retainAll(set);
+            return mutableSet;
+        }
+        else
+        {
+            Mutable<name>Set mutableSet = set.toSet();
```

Comment:
You won't need `toSet()` if you use `select`

Reply:
Done.